src/app/controllers/views/ControllerList/ControllerListTable/ControllerListTable.tsx

```diff
@@ -120,39 +117,33 @@ const generateRows = ({
                     }
                   />
                 ) : controller.vault_configured === true ? (
-                  unconfiguredControllers >= 1 && (
-                    <Tooltip
-                      children={
-                        <Icon
-                          aria-describedby="tooltip-description"
-                          data-testid="vault-icon"
-                          name="security"
-                        />
-                      }
-                      message={
-                        <p id="tooltip-description">
-                          Vault is configured on this controller. <br />
-                          Once all controllers are configured, migrate the
-                          secrets. <br />
-                          <a href={docsUrls.vaultIntegration}>
-                            Read more about Vault integration
-                          </a>
-                        </p>
-                      }
-                    />
-                  )
+                  <TooltipButton
+                    iconName="security"
+                    iconProps={{
+                      "data-testid": "vault-icon",
+                      "aria-describedby": "tooltip-description",
+                    }}
+                    message={
+                      <p id="tooltip-description">
+                        Vault is configured on this controller. <br />
+                        Once all controllers are configured, migrate the
+                        secrets. <br />
+                        <a href={docsUrls.vaultIntegration}>
+                          Read more about Vault integration
+                        </a>
+                      </p>
+                    }
+                  />
                 ) : (
                   configuredControllers >= 1 && (
-                    <Tooltip
-                      children={
-                        <Icon
-                          aria-describedby="tooltip-description"
-                          data-testid="vault-icon"
-                          name="security-warning"
-                        />
-                      }
+                    <TooltipButton
+                      iconName="security-warning"
+                      iconProps={{
+                        "data-testid": "vault-icon",
+                        "aria-describedby": "tooltip-description-warning",
+                      }}
                       message={
-                        <p id="tooltip-description">
+                        <p id="tooltip-description-warning">
                           Missing Vault configuration.
                           <br />
                           <a href={docsUrls.vaultIntegration}>
```

Comment:
Sorry, I missed this in the first review.

We have a special class to be used for links in tooltips (which have a dark background) to get accessibility compliant colour contrast. Also, external docs links are usually opened in a new tab. `className="is-on-dark"`.

Could you please update along with other links in this file.
 
```suggestion
                          <Link
                            href={docsUrls.vaultIntegration}
                            rel="noreferrer noopener"
                            target="_blank"
                          >
```